designer/server/src/main/scala/pl/touk/nussknacker/ui/config/scenariotoolbar/ToolbarButtonConfig.scala

```diff
@@ -2,13 +2,16 @@ package pl.touk.nussknacker.ui.config.scenariotoolbar
 
 import io.circe.{Decoder, Encoder}
 import pl.touk.nussknacker.ui.config.scenariotoolbar.ToolbarButtonConfigType.ToolbarButtonType
+import pl.touk.nussknacker.ui.process.DocsButtonConfig
 
 final case class ToolbarButtonConfig(
     `type`: ToolbarButtonType,
     name: Option[String],
     title: Option[String],
     icon: Option[String],
     url: Option[String],
+    markdownContent: Option[String],
+    docs: Option[DocsButtonConfig],
```

Comment:
move these 2 new args as the last ones with default values = None. This way you won't have to change tests and these values should be None by default imo as far as I understand them.